File: sis-kitnet/src/app/core/model/dto/indicadores/indicadoresResumoDTO.ts
export interface IndicadoresResumoDTO {
  // Apartamentos
  totalApartamentos: number;
  totalApartamentosAlugados: number;
  totalApartamentosVagos: number;
  
  // Financeiro
  receitaPrevista: number;
  totalRecebido: number;
  totalDebito: number;
  totalEmAberto: number;
  
  // Taxas (%)
  taxaOcupacao: number;
  taxaInadimplencia: number;
  taxaPagamento: number;
  
  // Controle
  totalPagamentos: number;
  pagamentosVencidos: number;
}

File: kitnet/src/main/java/kitnet/com/api/dto/indicadores/IndicadoresResumoDTO.java
package kitnet.com.api.dto.indicadores;

import java.math.BigDecimal;

public record IndicadoresResumoDTO(
    Long totalApartamentos,
    Long totalApartamentosAlugados,
    Long totalApartamentosVagos,
    BigDecimal receitaPrevista,
    BigDecimal totalRecebido,
    BigDecimal totalDebito,
    BigDecimal totalEmAberto,
    Double taxaOcupacao,
    Double taxaInadimplencia,
    Double taxaPagamento,
    Long totalPagamentos,
    Long pagamentosVencidos,
    BigDecimal somaAlugueisEmAberto,
    BigDecimal totalDebitoReal
) {}
